script/node/setup.bat

```diff
@@ -0,0 +1,3 @@
+setlocal
```

Comment:
Use `setlocal` to not leave the user in the `node` directory after running this script